plugins/vpp/natplugin/descriptor/nat44_vrf.go

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) 2018 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package descriptor
+
+import (
+	"go.ligato.io/cn-infra/v2/logging"
+	kvs "go.ligato.io/vpp-agent/v3/plugins/kvscheduler/api"
+	"go.ligato.io/vpp-agent/v3/plugins/vpp/natplugin/descriptor/adapter"
+	"go.ligato.io/vpp-agent/v3/plugins/vpp/natplugin/vppcalls"
+	nat "go.ligato.io/vpp-agent/v3/proto/ligato/vpp/nat"
+	"strconv"
+)
+
+const (
+	NAT44VrfDescriptorName = "vpp-nat44-vrf"
+
+	//Dependencies
+	nat44VrfTableDep = "vrf-table-exists"
+)
+
+// A list of non-retriable errors:
+var (
+// ErrNATInterfaceFeatureCollision is returned when VPP NAT features assigned
+// to a single interface collide.
+//ErrNATInterfaceFeatureCollision = errors.New("VPP NAT interface feature collision")
+
+// ErrDuplicateNATAddress is returned when VPP NAT address pool contains duplicate
+// IP addresses.
+// ErrDuplicateNATAddress = errors.New("Duplicate VPP NAT address")
+)
+
+// NAT44GlobalDescriptor teaches KVScheduler how to configure global options for
+// VPP NAT44.
+
+type NAT44VrfDescriptor struct {
+	log        logging.Logger
+	natHandler vppcalls.NatVppAPI
+
+	// UseDeprecatedAPI tracks whether deprecated global API (NAT interfaces, addresses) is being used on NB.
+	// Used to orchestrate which data should be dumped from which descriptor on Retrieve.
+	UseDeprecatedAPI bool
```

Comment:
The comment is not really up-to-date (another copy/paste). Do you need the bool variable itself?